website/versioned_docs/version-v3.10.x/externaldata.md

```diff
@@ -449,3 +449,71 @@ server := &http.Server{
 ```
 
 2. If `cert-controller` is disabled via the `--disable-cert-rotation` flag, you can use a cluster-wide, well-known CA certificate for Gatekeeper so that your external data provider can trust it without being deployed to the `gatekeeper-system` namespace.
+
+### Authenticate API server against Webhook (Self managed K8s cluster only)
+
+**Note:** To enable authenticate api server requires modifying cluster resources that may not be possible in managed K8s cluster
+
+To ensure a request to the Gatekeeper webhook is coming from the api server, Gatekeeper needs to validate the client cert in the request. To enable authenticate api server, the following configuration can be made:
+
+1. Deploy gatekeeper with a client CA cert name. Provide name of the client CA with the flag `--client-cert-name`. The same name will be used to read certificate from the webhook secret. The webhook will only authenticate apiserver requests if client CA name is provided with flag.
+
+2. You will need to patch the webhook secret manually to attach client ca crt. Update secret `gatekeeper-webhook-server-cert` to include `clientca.crt`. Key name `clientca.crt` should match the name passed with `--client-cert-name` flag.
```

Comment:
I should have asked this in the previous PR, should we include openssl cmds here to show how to generate clientca.crt?

Reply:
aren't we using apiserver's cert here so we don't want to generate?

Reply:
Yes we are! May be to make it unambiguous we could say `attach apiserver's client ca crt` 🤔

Reply:
@JaydipGabani sounds good